tests/robot-tests/tests/admin_and_public/bau/publish_publication_update.robot

```diff
@@ -0,0 +1,71 @@
+*** Settings ***
+Library             ../../libs/admin_api.py
+Resource            ../../libs/public-common.robot
+Resource            ../../libs/admin-common.robot
+
+Suite Setup         user signs in as bau1
+Suite Teardown      user closes the browser
+Test Setup          fail test fast if required
+
+Force Tags          Admin    Local    Dev    AltersData
+
+*** Variables ***
+${PUBLICATION_NAME}=            UI tests - publish publication update %{RUN_IDENTIFIER}
+${PUBLICATION_NAME_UPDATED}=    ${PUBLICATION_NAME} updated
+
+*** Test Cases ***
+Create Publication as bau1
+    ${PUBLICATION_ID}=    user creates test publication via api    ${PUBLICATION_NAME}
+    user create test release via api    ${PUBLICATION_ID}    AY    2046
+
+Publish release
+    user navigates to editable release summary from admin dashboard    ${PUBLICATION_NAME}
+    ...    Academic Year 2046/47 (not Live)
+    user approves release for immediate publication
+
+Navigate to publication on admin dashboard
+    ${accordion}=    user opens publication on the admin dashboard    ${PUBLICATION_NAME}
+    user clicks link    Manage this publication    ${accordion}
+    user waits until page contains title    Manage publication
+
+Update publication details
+    user enters text into element    id:publicationForm-title    ${PUBLICATION_NAME_UPDATED}
```

Comment:
Can use a keyword that accepts the formfield label here for better readability, I thiiiiink...